Dockerfile

```diff
@@ -27,16 +27,14 @@ ENV GO_LDFLAGS="-X ${PKG}/version.Version=${TAG}"
 RUN go-build-static.sh -gcflags=-trimpath=${GOPATH}/src -o bin/flanneld .
 RUN go-assert-static.sh bin/*
 RUN if [ "${ARCH}" != "s390x" ]; then \
-      go-assert-boring.sh bin/* ; \
+    go-assert-boring.sh bin/* ; \
     fi
 RUN install -s bin/* /usr/local/bin
 RUN flanneld --version
 
-FROM ubi
-RUN microdnf update -y          && \
-    microdnf install -y yum     && \
-    yum install -y ca-certificates \
-    strongswan net-tools which  && \
-    rm -rf /var/cache/yum
+FROM bci
+RUN zypper update -y && \
```

Comment:
Did you mean to update all the existing packages or refresh repository cache? In case of the latter it should be ```zypper ref```

Reply:
I wanted to update all things to the latest.